manim/mobject/geometry/polygram.py

```diff
@@ -72,20 +78,25 @@ def construct(self):
     """
 
     def __init__(
-        self, *vertex_groups: Point3D, color: ParsableManimColor = BLUE, **kwargs
+        self,
+        *vertex_groups: InternalPoint3D,
```

Comment:
Shouldn't this still be `Point3D`? The examples given in the documentation make it seem like tuples/lists are allowed...

Reply:
It works fine with `Point3D`. Implemented.